tri.css

```
@@ -1,47 +1,57 @@
-body{
+body {
   background: black;
   margin: 0;
   font-size: 100%;
 }
 
-h1{
+h1 {
   background: green;
   margin: 0;
   text-align: center;
   color: white;
   padding: 8px 0;
 }
 
-#titleDisplay{
+#titleDisplay {
   font-size: 150%;
 }
 
-#stripe{
+#stripe {
   background: black;
   height: 50px;
   margin: 0;
   padding-top: 10px;
 }
 
 .btn {
-  display:block;
+  display: block;
   height: 50px;
   width: 50px;
   border-radius: 50%;
   border: 6px solid black;
-  margin:0 auto;
-  background-image: url("assets/start.jpg"); 
-  background-repeat:no-repeat;
+  margin: 0 auto;
+  background-image: url("assets/start.jpg");
+  background-repeat: no-repeat;
   background-size: 140%;
   background-position: -8px -8px;
 }
 
-#reset{
-
-}
 
-#one:hover, #two:hover, #three:hover, #four:hover, #five:hover, #six:hover, #seven:hover, #eight:hover, #nine:hover,
-#ten:hover, #eleven:hover, #twelve:hover, #thirteen:hover, #fourteen:hover, #fifteen:hover { 
+#one:hover,
+#two:hover,
+#three:hover,
+#four:hover,
+#five:hover,
+#six:hover,
+#seven:hover,
+#eight:hover,
+#nine:hover,
+#ten:hover,
+#eleven:hover,
+#twelve:hover,
+#thirteen:hover,
+#fourteen:hover,
+#fifteen:hover {
```

Instead of selecting each Id, you could use the class name as a selector
```suggestion
.circle:hover {
```

---

Swapped class name for multiple id.